srcpkgs/dbeaver/files/dbeaver.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=DBeaver
```

Comment:
could you ask them to include this .desktop file upstream so every distribution doesn't have to include it on their own?

Reply:
https://github.com/dbeaver/dbeaver/issues/19174